### ISSUE TITLE
one2one/parent models' fields are not serialized

### ISSUE THREAD

**Issue by nerdoc** [adamghill/django-unicorn#459]
I have a model
```python
# models.py
from django.contrib.sites.models import Site

class Homepage(Site):
    subtitle = models.CharField(max_length=255,...)
    logo = models.ImageField()
    theme = models.ForeignKey(Theme, default=1, on_delete=models.SET_DEFAULT)
```

When using that in a UnicornView:
```python
# components/master_data.py
from ..models import Homepage

class MasterDataView(UnicornView):
    homepage: Homepage = None

    def save(self):
        self.homepage.save()
```
and try to get that into HTML:
```html
<!-- master_data.html -->
  <div>
    <input unicorn:model.defer="homepage.name" type="text" />
    {{ homepage.name }}
    <input unicorn:model.defer="homepage.subtitle" type="text" />
    {{ homepage.subtitle }}
    <button u:prevent unicorn:click="save()">Save</button>
  </div>
```
I don't get the  `homepage.name` value - which should be inherited from `Site`. If I look at the js data Unicorn transfers:
```html
<script type="application/json" id="unicorn:data:4DTnYCxq">{"id":"4DTnYCxq","name":"homepage.master_data","key":"","data":{"homepage":{"subtitle":"A subtitle!!","logo":"","theme":1,"pk":3},"view_is_async":false},"calls":[],"hash":"aTPpLzxk"}</script>
```

I think this should not be the case, as I would expect that `Site.domain` and `Site.name` is serialized too.

**Comment by adamghill** [comment]
Thanks for making this issue, I'll take a look!

**Comment by adamghill** [comment]
I use `serializers.serializer` for Django models so [this](https://docs.djangoproject.com/en/4.1/topics/serialization/#inherited-models) is _why_ it's happening. I just am not sure what I am going to do to get around the limitation. I have a few ideas, but nothing solid, yet.

**Comment by adamghill** [comment]
https://github.com/adamghill/django-unicorn/pull/468 has been merged into main and is included in the new 0.49.0 version. Closing this for now, but let me know if you run into any issues with this going forward.
